DFC.App.RelatedCareers/AutoMapperProfiles/RelatedCareersSegmentModelProfile.cs

```diff
@@ -1,17 +1,16 @@
 ﻿using AutoMapper;
 using DFC.App.RelatedCareers.Data.Models;
 using DFC.App.RelatedCareers.ViewModels;
-using Microsoft.AspNetCore.Html;
 
```

Comment:
The content property should be a htmlstring so automapper will need to map this. Check this

Reply:
This model does not have a Content property, only Data